Sources/BraintreePayPal/BTPayPalRequest.swift

```diff
@@ -49,7 +49,7 @@ import BraintreeCore
 
 /// Base options for PayPal Checkout and PayPal Vault flows.
 /// - Note: Do not instantiate this class directly. Instead, use BTPayPalCheckoutRequest or BTPayPalVaultRequest.
-@objcMembers public class BTPayPalRequest: NSObject {
+@objcMembers open class BTPayPalRequest: NSObject {
```

Comment:
This is now needs to be `open` because we are subclassing it outside of it's defining module